python/mrchem/update_input_parser.py

```diff
@@ -71,6 +71,139 @@ def update_constants():
     yaml.dump(new, f_template)
 
 
+def update_periodic_table():
+    f_template = root.joinpath("template.yml")
+    template = yaml.load(f_template)
+
+    new = {
+        "keywords": template["keywords"],
+        "sections": [section for section in template["sections"] if section["name"] != "Elements"],
+    }
+
+    # Build new Periodic table Section
+    Elements = {"name": "Elements", "sections": [], "docstring": "list of elements with data"}
+    for ele, vals in pt.PeriodicTableByName.items():
+        if (ele.lower() == "none") or ((ele.lower() == "x") or (ele.lower() == "q")):
+            continue
+        elif ele.lower() == "no":
+            # FIXME this is a hack to get around the fact that "no" is a reserved keyword in YAML
+            name = '"no"'
+            symbol = '"No"'
+        else:
+            name = str(ele)
+            symbol = vals[4]
+
+        context = qcel.VanderWaalsRadii("MANTINA2009")
+
+        if ele.lower() == "h":
```

Comment:
Adding a comment as to why we patch the value for Hydrogen might be useful for who reads this code in the future.
```suggestion
        if ele.lower() == "h":
           # these vdw radii are used with a continuum solvation model and 1.2 AA for hydrogen is more appropriate than the default
```